WordPress/Classes/ViewRelated/Tools/Time Zone/TimeZoneSelectorViewController.swift

```diff
@@ -65,22 +65,79 @@ struct TimeZoneSelectorViewModel: Observable {
         })
     }
 
+    private let timeZoneOffsetFormatter = DateFormatter()
+
+    private let timeAtTimeZoneFormatter = DateFormatter()
+
+    private let currentDate = Date()
+
+    // DateFormatter is expensive https://sarunw.com/posts/how-expensive-is-dateformatter/
+    // initializer to ensure that:
+    // 1. Existing 3 property struct initialization of state, selectedValue, filter work normally
+    // 2. DateFormatters are configured whenever we create instance of TimeZoneSelectorViewModel
+    // 3. DateFormatters are private and their state not mutable from outside callers (thread-safety)
```

Comment:
I don't think these comments should be here, and structured like this, for the following reasons:
1. this is the place where normally methods documentation should go (and, if so, it should be preceded by /// and not //)
2. DateFormatter is expensive: that's commonly known, no need to add a specific link to a web page that could change at any time
3. "DateFormatters are private and their state not mutable from outside callers (thread-safety)": properties that do not need to be accessed externally should be private regardless, and also, from the [documentation](https://developer.apple.com/documentation/foundation/nsdateformatter): "On iOS 7 and later NSDateFormatter is thread safe"

Reply:
No problem; I thought it was better to explain than not.  Removed!